frontend/src/components/Sidebar/SidebarItem.tsx

```diff
@@ -135,13 +135,15 @@ export default function SidebarItem(props: SidebarItemProps) {
         {...other}
       />
       {subList.length > 0 && (
-        <Collapse in={fullWidth && expanded}>
-          <List component="div" disablePadding className={classes.nested}>
-            {subList.map((item, i) => (
-              <SidebarItem key={i} selectedName={selectedName} hasParent {...item} />
-            ))}
-          </List>
-        </Collapse>
+        <li>
```

Comment:
I think it's important to use Material UIs abstractions as much as possible.
Would a [ListItem](https://material-ui.com/api/list-item/) work here?